src/main/scala/io/razem/influxdbclient/Database.scala

```diff
@@ -12,12 +14,13 @@ class Database protected[influxdbclient]
   with RetentionPolicyManagement
   with DatabaseManagement
 {
-  def write(point: Point,
+
+  def write[A](metric: A,
```

Comment:
I think we should not care about binary compatibility this time because artifact is being published under another group